check_lazy.py

```diff
@@ -61,6 +61,7 @@ def sweep_models(output_filename, tests=['eval', 'train']):
                     print(f"{datetime.strptime(dt,'%Y-%m-%d %H:%M:%S')} : Running launch_command {' '.join(launch_command)}")
                     rc = subprocess.call(launch_command,
                                         env=env,
+                                        timeout = 1800, # 30 minutes, 10 min max per iter
```

Comment:
great idea.  although.. why so long even, is there any model that should take more than a few min per iter?  I suspect we have some kind of unsolved issue with process startup or something.